### PR TITLE
Added the 'Adder' example

### PR DESCRIPTION
**Pull request by alshdavid**
Added this example application, showing how to obtain a value from some `Entry` fields and update a `Label` from an action.

example

### CODE REVIEW

**Comment by alshdavid** [comment]
Few basic questions (I am actively reading the documentation so I apologise if this is already covered there):

* How do I add padding to the window?
* How do I add padding to the bottom row?
* How do I react to the user hitting the `enter` keyboard key?

**Comment by diamondburned** [comment]
> * How do I add padding to the window?
> 
> * How do I add padding to the bottom row?

You can either use the `SetMargin{Top,Bottom,Start,End}` methods, or you can
give the application a stylesheet and theme it using classes.

I haven't made a CSS stylesheet example yet, but I probably should.

> 
> * How do I react to the user hitting the `enter` keyboard key?
>

Specifically within the input, you can do `input.Connect("activate", func()
{})`.

**Comment by diamondburned** [comment]
As a side note, I've created the Discussions page for any future questions that
anybody may have regarding the use of GTK4 in Go that isn't related to
the PR.

**Comment by diamondburned** [comment]
CSS example added in [cssdemo](https://github.com/diamondburned/gotk4-examples/tree/master/gtk4/cssdemo).

**Comment by alshdavid** [comment]
Thanks, please rereview

File: gtk4/adder/main.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	plusTitle := gtk.NewLabel("+")
+	plusTitle.SetMarginStart(8)
+	plusTitle.SetMarginEnd(8)
+	grid.Attach(plusTitle, 1, 0, 1, 1)
+
+	number2 := gtk.NewEntry()
+	number2.SetPlaceholderText("Enter Number")
+	grid.Attach(number2, 2, 0, 1, 1)
+
+	calculate := gtk.NewButtonWithLabel("Caluclate")
```
Comment (diamondburned) [inline gtk4/adder/main.go:44]: There's a typo here.

**Comment by diamondburned** [comment]
You can put the screenshot outside of `.docs` as well, no need for a new folder.

**Pull request closed without merging**